packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -2,35 +2,36 @@
 
 import { Typography } from "@/src/ui/typography";
 import { useCampaigns } from "@/src/hooks/useCampaigns";
-import { useState } from "react";
 import { useAvailableAmms } from "@/src/hooks/useAvailableAmms";
 import { Campaign } from "./campaign";
 
 import styles from "./styles.module.css";
-
-const PAGE_SIZE = 10;
+import { useTranslations } from "next-intl";
 
 export function Campaigns() {
-    const [pageNumber, setPageNumber] = useState(1);
+    const t = useTranslations();
```

Comment:
We can add `allCampaigns` as a param to this hook to have all the local `t` scoped under that key.

Reply:
Right, addressed in 29e717d56185787009a3d86f3a1966ed97f8c112